pkg/ibm/vpn.go

```diff
@@ -434,6 +434,7 @@ func (c *CloudClient) DeleteVPN(VPNGatewayID string) error {
 			utils.Log.Printf("Failed to delete routes of VPN connection %v, during VPN deletion process, with error: %+v", *connection.ID, err)
 			return err
 		}
+		time.Sleep(10 * time.Second)
```

Comment:
@praveingk Consider replacing sleep with `pollVPNStatus(ID, False)`.
It returns true if the VPN is ready/deleted within the alloted time frame.

Reply:
Thanks Omer. I will try that, I am currently hitting some race conditions during deletion. I think polling status would help.